src/app/screens/sendBrc20/index.tsx

```diff
@@ -96,12 +96,12 @@ function SendBrc20Screen() {
     }
   };
 
-  const checkIfEnableButton = () => {
+  const checkIfEnableButton = useMemo(() => {
     if (amountToSend !== '' || amountToSend <= fungibleToken.balance) {
       return true;
     }
     return false;
-  };
+  }, [amountToSend, fungibleToken]);
```

Comment:
this is not really an expensive calculation worth memoizing. since amountToSend is component state, it will already trigger a re-render, so you could simplify to:

`const isSendEnabled = !!(amountToSend !== '' || amountToSend <= fungibleToken.balance);`

Reply:
on closer look, the existing logic is broken. this should have been

`const isSendEnabled = amountToSend !== '' && amountToSend <= fungibleToken.balance`

but also the button doesn't get disabled, it just gets an opacity change. should fix by using the `disabled` prop on ActionButton